index.html

```diff
@@ -12,32 +12,187 @@
+            <button class="button about-button" type="button">Get My Resume</button>
+        </div>
+         <div><img src="images/Vector4.png" alt="line logo" class="line"></div>
+            <ul class="language">
+            <li><img src="images/Rectangle55.png" alt="my logo" class="img">Language
+                <ul>
+                    <li class="language-list">Javascript</li>
+                    <li class="language-list">Ruby</li>
+                    <li class="language-list">Html</li>
+                    <li class="language-list">css</li>
+                </ul>
+            </li>
+            <li>
+                <img src="images/Rectangle56.png" alt="logo" class="img" />Framework
+                <ul>
+                    <li class="language-list">bootstrap</li>
+                    <li class="language-list">Ruby on Rails</li>
+                    <li class="language-list">Capybara</li>
+                    <li class="language-list">selenium</li>
+                </ul>
+            </li>
+            <li>
+                <img src="images/rectangle57.png" alt="logo" class="img"/> Skills
+                <ul>
+                    <li class="language-list">codekit</li>
+                    <li class="language-list">Github</li>
+                    <li class="language-list">Gitlab</li>
+                    <li class="language-list">Terminal</li>
+                </ul>
+            </li>
+        </ul>
+    </section>
+    <section>
+        <h3 class="intro-text">I am always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <div class="container"></div>
+        <form class="form" action="https://formspree.io/f/xzbokwoy" method="post">
+            <input type="text" id="fname" name="name" placeholder="Full Name" maxlength="30" required>
+            <input id="email" name="email" type="email" placeholder="Email Address" required>
+            <textarea class="textarea" name="message" maxlength="500" placeholder="text" required>Enter text here
+            </textarea>
+            <button class="get-button" type="submit">Get In Touch</button>
+        </form>
+        <ul class="social-icons">
+            <li><img src="images/vector.png3.png" class="social-logos" alt="my logo"></li>
+            <li><img src="images/vector.png" class="social-logo logons" alt="my logo"></li>
+            <li><img src="images/vector.png8.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png4.png" alt="my logo" class="social-logo"></li>
+            <li><img src="images/vector.png2.png" alt="my logo" class="social-logo"></li>
+        </ul>
+    </section>
+   </main>
+<footer>
+    <div class="footer">
+      <p>@Copyright <a href="https://twitter.com/Ademibowale1/">Hycrown </a> 2022- All Right Reserved.</p>  
+    </div>
+  </footer>
```

Comment:
### ⚠️All Optional changes below I marked as optional because the previous reviewer forgot to mention them.⚠️

- [ ] [Optional] Please remove the `div` and `p` tags lines 193-195 because it's not in the given Figma design and add the social icons you placed in the contact form section here in the footer to match the given design and for semantic reasons.